symbology/render-multilayer-symbols/src/main/java/com/esri/samples/render_multilayer_symbols/RenderMultilayerSymbolsSample.java

```diff
@@ -0,0 +1,459 @@
+    var multilayerPolylineSymbol = new MultilayerPolylineSymbol(List.of(solidStrokeSymbolLayer));
+
+    // create a polyline graphic with geometry using the symbol created above, and add it to the graphics overlay
+    graphicsOverlay.getGraphics().add(new Graphic(polylineBuilder.toGeometry(), multilayerPolylineSymbol));
+  }
+
+  /**
+   * Adds new graphics constructed from multilayer polygon symbols.
+   *
+   * @param angles A list containing the angle at which to draw any fill lines within the polygon.
+   * @param offset The value used to keep a consistent distance between symbols in the same column.
+   */
+  private void addPolygonGraphicsWithMarkerSymbols(List<Double> angles, double offset) {
+    var polygonBuilder = new PolygonBuilder(SpatialReferences.getWgs84());
+    polygonBuilder.addPoint(new Point(60, 25-offset));
+    polygonBuilder.addPoint(new Point(70, 25-offset));
+    polygonBuilder.addPoint(new Point(70, 20-offset));
+    polygonBuilder.addPoint(new Point(60, 20-offset));
+
+    // create a stroke symbol layer to be used by patterns
+    SolidStrokeSymbolLayer strokeForHatches = new SolidStrokeSymbolLayer(2, ColorUtil.colorToArgb(Color.RED),
+      List.of(new DashGeometricEffect()));
+
+    // create a stroke symbol layer to be used as an outline for aforementioned patterns
+    SolidStrokeSymbolLayer strokeForOutline = new SolidStrokeSymbolLayer(1, ColorUtil.colorToArgb(Color.BLACK),
+      List.of(new DashGeometricEffect()));
+
+    var symbolLayerArray = new SymbolLayer[angles.size() + 1];
+    for (int i = 0; i < angles.size(); i++) {
+      var hatchFillSymbolLayer = new HatchFillSymbolLayer(new MultilayerPolylineSymbol(List.of(strokeForHatches)), angles.get(i));
+
+      // define separation distance for lines and add them to the symbol layer list
+      hatchFillSymbolLayer.setSeparation(9);
+      symbolLayerArray[i] = hatchFillSymbolLayer;
+    }
+    symbolLayerArray[symbolLayerArray.length - 1] = strokeForOutline;
+    // create a multilayer polygon symbol with symbol layers
+    var multilayerPolygonSymbol = new MultilayerPolygonSymbol(List.of(symbolLayerArray));
+
+    // create a polygon graphic with geometry using the symbol created above, and add it to the graphics overlay
+    var graphic = new Graphic(polygonBuilder.toGeometry(), multilayerPolygonSymbol);
+    graphicsOverlay.getGraphics().add(graphic);
+  }
+
+  /**
+   * Creates a complex point from multiple symbol layers and a provided geometry.
+   * @param complexPointGeometry A base geometry upon which other symbol layers are drawn.
+   */
+  private void addComplexPoint(Geometry complexPointGeometry) {
+    // create marker layers for complex point
+    var orangeSquareVectorMarkerLayer = getLayerForComplexPoint(Color.ORANGE, Color.BLUE, 11);
+    var blackSquareVectorMarkerLayer = getLayerForComplexPoint(Color.BLACK, Color.ORANGERED, 6);
+    var purpleSquareVectorMarkerLayer = getLayerForComplexPoint(Color.TRANSPARENT, Color.PURPLE, 14);
```

Comment:
since it's not super clear here what type the var's are, let's use the full declaration